cocotb/regression.py

```diff
@@ -115,7 +115,11 @@ def __init__(self, dut: SimHandle, tests: Iterable[Test], hooks: Iterable[Hook])
 
         if coverage is not None:
             self.log.info("Enabling coverage collection of Python code")
-            self._cov = coverage.coverage(branch=True, omit=["*cocotb*"])
+            # TODO: We want to exclude cocotb code form coverage collection and
+            # only collect coverage for user code. Using omit as shown below is
+            # a hackish workaround to exclude installed packages, but it's
+            # nowhere near fool-proof.
+            self._cov = coverage.coverage(branch=True, omit=["*site-packages*"])
```

Comment:
Can we be more specific to not ignore installed user packages?
```suggestion
            self._cov = coverage.coverage(branch=True, omit=["*site-packages/cocotb/*"])
```
I don't know if this needs `\` instead of `/` on windows.

Reply:
Done. (I'm not sure which approach would be more useful in the end, but only stripping out cocotb is certainly what was intended before, so keeping that behavior makes sense.)

Forward slashes are also used on Windows, see https://github.com/nedbat/coveragepy/blob/225dd7cd63336b90c6e0b55fbe800a9241b3695b/coverage/files.py#L297.